ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/textinput.html

```diff
@@ -47,6 +49,8 @@
            aria-label="${text.label.visible && text.label.value}"
            type="text"
            name="${text.name}"
+           data-cmp-data-layer="${text.data.json}"
+           data-cmp-clickable="${text.data ? true : false}"
```

Comment:
What data-cmp-clickable mean ? What does `text.data` return ?

Reply:
it sends an event when this element is clicked (as used by sites)

Reply:
I added this for field visit events. Since I have field visit event on guidebridge, I no longer need this. Should I remove the same?

Reply:
yes, we can remove clickable from all form components (except title, header, footer)

Reply:
removed.